PWGHF/D2H/Tasks/taskDstarToD0Pi.cxx

```diff
@@ -267,8 +267,8 @@ struct HfTaskDstarToD0Pi {
       // Retrieve the file from CCDB
       bool isFileAvailable = ccdbApi.retrieveBlob(ccdbPathForWeight, ".", metadata, timestampCCDB, false, weightFileName);
       if (!isFileAvailable) {
-        LOGF(error, "Failed to retrieve weight file from CCDB: %s", ccdbPathForWeight.value.c_str());
-        return;
+        LOGF(fatal, "Failed to retrieve weight file from CCDB: %s", ccdbPathForWeight.value.c_str());
+        abort();
```

Comment:
`abort` is not needed. The `fatal` level already sends the abort signal to the steering process. Just use `return` after.